### ISSUE TITLE
Add option to set global default language parameter

### ISSUE THREAD

**Issue by ErenKrkl** [RatakondalaArun/tmdb_api#81]
At the time when we call the "api", it can be a very nice feature if a parameter related to changing the language was added. So I can add options related to changing the language to my application's settings. 

**For example;**
```dart
final String apikey = 'my api key';
final String readaccesstoken = 'my readaccesstoken';
final String languages = 'tr-TR';

TMDB tmdbWithCustomLogs = TMDB(
      ApiKeys(apikey, readaccesstoken, languages),
      logConfig: ConfigLogger(
        showLogs: true,
        showErrorLogs: true,
      ),
    );
```
I just tried to give an example to make it a little more clear what I'm saying. I would be very grateful if you take it into account. Have a nice work day.

**Comment by RatakondalaArun** [comment]
@ErenKrkl 

🚀Released in [v2.1.1](https://pub.dev/packages/tmdb_api)

**Comment by ErenKrkl** [comment]
Hello again. Am I doing something wrong? The api language still comes in English even though I added the newly added language parameter. Am I making an incorrect or incomplete use? I'm new to these topics, so excuse me. I am adding my screen output. From: Ratakondala ArunSent: Sunday, April 24, 2022 10:13 AMTo: RatakondalaArun/tmdb_apiCc: Eren Kırkıl; MentionSubject: Re: [RatakondalaArun/tmdb_api] Add option to set global default language parameter (Issue ***@***.***🚀Released in v2.1.1—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by RatakondalaArun** [comment]
@ErenKrkl please share the method that is returning wrong data
